src/main/java/fr/insee/pearljam/api/dto/campaign/CampaignDto.java

```diff
@@ -74,10 +74,10 @@ public CampaignDto(String id, String label, String email, IdentificationConfigur
 
 	public void setCampaignStats(List<Object[]> obj) {
 		if (obj != null && !obj.isEmpty() && obj.get(0).length > 3 && obj.get(0)[0] != null) {
```

Comment:
strange

Reply:
too much output cases and too few dto
better archi is the answer, to be continued